packages/issuer/lib/__tests__/VcIssuer.spec.ts

```diff
@@ -12,7 +12,8 @@ import {
   ProofOfPossession,
   Typ,
 } from '@sphereon/openid4vci-common'
-import { IProofPurpose, IProofType } from '@sphereon/ssi-types'
+import { generateDid, getIssuerCallback } from '@sphereon/openid4vci-issuer-callback/dist/IssuerCallback'
```

Comment:
Import is not correct. Also not sure why the issuer lib would depend on the issuer-callback lib instead of the other way around